src/fidesops/ops/api/v1/endpoints/privacy_request_endpoints.py

```diff
@@ -283,27 +284,64 @@ def privacy_request_csv_download(
     return response
 
 
-def execution_logs_by_dataset_name(
+def execution_and_audit_logs_by_dataset_name(
     self: PrivacyRequest,
 ) -> DefaultDict[str, List["ExecutionLog"]]:
     """
-    Returns a truncated list of ExecutionLogs for each dataset name associated with
-    a PrivacyRequest. Added as a conditional property to the PrivacyRequest class at runtime to
-    show optionally embedded execution logs.
+    Returns a combined mapping of execution and audit logs for the given privacy request.
+
+    Audit Logs are for the entire privacy request as a whole, while execution logs are created for specific collections.
+    Logs here are grouped by dataset, but if it is an audit log, it is just given a fake dataset name, here "Request + status"
+    ExecutionLogs for each dataset are truncated.
+
+    Added as a conditional property to the PrivacyRequest class at runtime to
+    show optionally embedded execution and audit logs.
 
     An example response might include your execution logs from your mongo db in one group, and execution logs from
-    your postgres db in a different group.
+    your postgres db in a different group, plus audit logs for when the request was approved and denied.
     """
+    db: Session = Session.object_session(self)
+    all_logs: DefaultDict[str, List[Union["AuditLog", "ExecutionLog"]]] = defaultdict(
+        list
+    )
 
-    execution_logs: DefaultDict[str, List["ExecutionLog"]] = defaultdict(list)
-
-    for log in self.execution_logs.order_by(
-        ExecutionLog.dataset_name, ExecutionLog.updated_at.asc()
-    ):
-        if len(execution_logs[log.dataset_name]) > EMBEDDED_EXECUTION_LOG_LIMIT - 1:
+    execution_log_query: Query = db.query(
+        ExecutionLog.id,
+        ExecutionLog.created_at,
+        ExecutionLog.updated_at,
+        ExecutionLog.message,
+        cast(ExecutionLog.status, sqlalchemy.String).label("status"),
+        ExecutionLog.privacy_request_id,
+        ExecutionLog.dataset_name,
+        ExecutionLog.collection_name,
+        ExecutionLog.fields_affected,
+        ExecutionLog.action_type,
+        null().label("user_id"),
+    ).filter(ExecutionLog.privacy_request_id == self.id)
+
+    audit_log_query: Query = db.query(
+        AuditLog.id,
+        AuditLog.created_at,
+        AuditLog.updated_at,
+        AuditLog.message,
+        cast(AuditLog.action.label("status"), sqlalchemy.String).label("status"),
+        AuditLog.privacy_request_id,
+        null().label("dataset_name"),
+        null().label("collection_name"),
+        null().label("fields_affected"),
+        null().label("action_type"),
+        AuditLog.user_id,
+    ).filter(AuditLog.privacy_request_id == self.id)
+
+    combined: Query = execution_log_query.union_all(audit_log_query)
```

Comment:
It looks like `union all` keeps all of the records from both original tables, but union removes duplicates.  union sorts and eliminates duplicates first.  In early testing, `union` was returning too many records, but in hindsight this may have had nothing to do with the `union` and may have been related to the fact that I didn't have other parts of my query correct yet.

Regardless, I think union all makes sense here, we don't need to do the extra step of sorting and dupe-removal.